content/spin/v2/rust-components.md

```diff
@@ -373,20 +367,19 @@ Spin has a key-value store built in. For information about using it from Rust, s
 
 The Spin key-value API stores and retrieves only lists of bytes. The Rust SDK provides helper functions that allow you to store and retrieve [Serde](https://docs.rs/serde/latest/serde) serializable values in a typed way. The underlying storage format is JSON (and is accessed via the `get_json` and `set_json` helpers). 
 
-To use the `get_json` and `set_json` helpers, you must enable the Spin SDK's optional `json` feature in your `Cargo.toml` file. To make your objects serializable, you will also need a reference to `serde`. The relevant `Cargo.toml` entries look like this:
+To make your objects serializable, you will also need a reference to `serde`. The relevant `Cargo.toml` entries look like this:
```

Comment:
This appears to have lost some information - unless we now unconditionally bring `serde` in with the SDK, which would be a bit surprising...?

Reply:
We now by default have the `get_json` and `set_json` helpers. They are still behind a feature flag but that flag is on by default.

Reply:
Aha!  I did not know that - seems like a good solution!